src/Storages/StorageMaterializedView.cpp

```diff
@@ -91,6 +91,12 @@ StorageMaterializedView::StorageMaterializedView(
 {
     StorageInMemoryMetadata storage_metadata;
     storage_metadata.setColumns(columns_);
+    auto * storage_def = query.storage;
```

Comment:
probably just use `auto` instead of `auto *` to simplify the code here

Reply:
Interestingly, I had `auto storage_def` initially, but some style check complained and suggested to change it to `auto *` to make it more explicit.

Reply:
Ah, I see, then just use what the style check suggests.